### ISSUE TITLE
Consider using self:GetWidth instead of self:GetSize in Display:CalculateSizeRatio

### ISSUE THREAD

**Issue by vomaufgang** [tullamods/tullaCC#22]
Back in version 7.2.0 the font size for any given cooldown was calculated based on the width of the icon it was displayed on:

~~~ lua
function Timer.OnSizeChanged(self, width, height)
	local fontScale = round(width) / ICON_SIZE
~~~

ad0dff93341718722c982979809bc44ec785c660 changed the calculation to [use the icons size via GetSize](https://github.com/tullamods/tullaCC/commit/ad0dff93341718722c982979809bc44ec785c660#diff-50c739240a5f7b52fd2723a13febc0b9R96-R97) instead:

~~~ lua
function Display:CalculateSizeRatio()
    local sizeRatio = round(min(self:GetSize())) / ICON_SIZE
~~~

GetSize however causes the calculation to use the icons **height** instead.

The cooldown icons above the floating enemy health bars are much wider than they are high - in my case 20 wide by 14 high.

The old calculation using width resulted in a sizeRatio of ca. 20 / 36 = 0.55, which was quite readable:

![grafik](https://user-images.githubusercontent.com/8030636/93737124-a05f4400-fbe2-11ea-820e-80c40e6159e6.png)

The new calculation however leads to a sizeRatio of ca. 14 / 36 = 0.38, making the font way too small imho:

![grafik](https://user-images.githubusercontent.com/8030636/93737288-0d72d980-fbe3-11ea-896e-7af714ad7805.png)

Would it be possible to change the calculation so the icons width / `self.GetWidth()` is used to determine the font size (again)?

Square icons would be unaffected by this change and *to my knowledge* there are no other non square icons with cooldowns in the game.

Thanks in advance for your time!

**Comment by Tuller** [comment]
You probably just want to switch min to max.

**Comment by vomaufgang** [comment]
You're right, didn't even see that `min`. Talk about standing in the forest being blind to the trees.
